packages/ui/components/dao/DaoCard.tsx

```diff
@@ -1,5 +1,129 @@
-export interface DaoCardProps {}
+import clsx from 'clsx'
+import Link from 'next/link'
+import { useTranslation } from 'react-i18next'
 
-export const DaoCard = ({}: DaoCardProps) => {
-  return <p>placeholder</p>
+import {
+  DaoCardMemberCheck,
+  DaoCardPin,
+  DaoCardProposals,
+  Governance,
+  SubDaoArrow,
+} from '@dao-dao/icons'
+
+export interface SubDaoInfo {
+  parentDaoHref: string
+  parentDaoImageUrl: string
+}
+
+export interface DaoCardProps {
+  name: string
+  description: string
+  imageUrl: string
+  established: Date
+  href: string
+
+  junoBalance: number
+  proposalCount: number
+
+  subDaoInfo?: SubDaoInfo
+
+  pinned: boolean
+  onPin: () => void
+}
+
+export const DaoCard = ({
+  name,
+  description,
+  imageUrl,
+  href,
+  established,
+  junoBalance,
+  proposalCount,
+  pinned,
+  onPin,
+  subDaoInfo,
+}: DaoCardProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <Link href={href} passHref>
+      <a className="flex relative flex-col justify-between items-center py-7 px-6 w-[260px] h-[328px] bg-background-secondary hover:bg-background-interactive-hover active:bg-background-interactive-pressed rounded-md outline-transparent hover:outline-border-interactive-hover active:outline-border-interactive-focus outline transition-all">
```

Comment:
I think we should make these flexibly sized like I did in [this PR](https://github.com/DA0-DA0/dao-dao-ui/pull/781), setting the max size instead (so it looks good in storybook) and letting the container set the card sizes responsively like this: https://github.com/DA0-DA0/dao-dao-ui/pull/781/files#diff-f1323ac505364a0f34805705604fde89a0376fcf45811061770dfcd9e0c21816R9-R21

Reply:
Am down. Would like to try both later to see what looks best. :) I really like the dimensions that edamame is using here.

Reply:
Yeah and we should for sure aim to get these dimensions roughly, by setting the container columns responsively like in that PR, but if we make them constant then we'll get the same weird home page sizing stuff we got a month ago.